src/constants/dexes.ts

```diff
@@ -1,380 +1,17 @@
-  zipswap: {
-    name: 'ZipSwap',
-    icon: 'https://zipswap.fi/static/media/logo.414dd3f8.png',
-  },
-  '1inch': {
-    name: '1inch',
-    icon: 'https://s2.coinmarketcap.com/static/img/coins/64x64/8104.png',
-  },
-  platypus: {
-    name: 'Platypus',
-    icon: 'https://s2.coinmarketcap.com/static/img/exchanges/64x64/1674.png',
-  },
-  velodrome: {
-    name: 'Velodrome',
-    icon: 'https://s2.coinmarketcap.com/static/img/coins/64x64/20435.png',
-  },
-}
-
-type DexTypes = {
-  [chainId in ChainId | 'all']?: {
-    [dex: string]: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9
-  }
-}
-/*
-// dex id - swap fee
-1 - 30 (default) = 0.3%
-2 - 25
-3 - 20
-4 - 15
-5 - 10
-6 - 5
-7 - 17 = 0.17%
-8 - 18 = 0.18%
-9 - 50 = 0.5%
-*/
-export const dexIds: DexTypes = {
-  all: {
-    firebird: 1,
-    pancake: 2,
-    'pancake-legacy': 2,
-    apeswap: 3,
-    wault: 3,
-    biswap: 5,
-    polydex: 5,
-    jetswap: 5,
-    polycat: 2,
-    spookyswap: 3,
-    axial: 3,
-    cronaswap: 2,
-    gravity: 2,
-    kyberswap: 0,
-    mmf: 7,
-    kryptodex: 3,
-    cometh: 9,
-    dinoswap: 8,
-    safeswap: 2,
-    pantherswap: 3,
-    morpheus: 4,
-    wannaswap: 3,
-    swapr: 2,
-    wagyuswap: 3,
-    astroswap: 3,
-  },
-  [ChainId.BSCMAINNET]: {
-    jetswap: 1,
-  },
-  [ChainId.MATIC]: {},
-}
-
-// For encoding
-export const dexTypes: DexTypes = {
-  all: {
-    curve: 2,
-    dmm: 3,
-    kyberswap: 3,
-    oneswap: 1,
-    ellipsis: 2,
-    nerve: 1,
-    'iron-stable': 4,
-    balancer: 6,
-    synapse: 4,
-    saddle: 4,
-    axial: 4,
-    beethovenx: 6,
-    uniswapv3: 5,
-    kyberswapv2: 5,
-    kyberswapStatic: 3,
-  },
-  [ChainId.MAINNET]: {},
-  [ChainId.BSCMAINNET]: {},
-  [ChainId.MATIC]: {},
-  [ChainId.AVAXMAINNET]: {},
-  [ChainId.FANTOM]: {},
-  [ChainId.VELAS]: {},
-  [ChainId.OASIS]: {},
-}
-
-function findDex(exchange: string): DexConfig | undefined {
-  const dex = dexListConfig[exchange]
-  return dex ? { ...dex, value: exchange } : undefined
-}
-
-export const DEX_TO_COMPARE: { [chainId in ChainId]?: DexConfig } = {
-  [ChainId.BSCMAINNET]: findDex('pancake'),
-  [ChainId.MATIC]: findDex('quickswap'),
-  [ChainId.AVAXMAINNET]: findDex('traderjoe'),
-  [ChainId.MAINNET]: findDex('uniswapv3'),
-  [ChainId.FANTOM]: findDex('spookyswap'),
-  [ChainId.CRONOS]: findDex('vvs'),
-  [ChainId.AURORA]: findDex('trisolaris'),
-  [ChainId.ARBITRUM]: findDex('sushiswap'),
-  [ChainId.VELAS]: findDex('wagyuswap'),
-  [ChainId.OASIS]: findDex('valleyswap-v2'),
-  [ChainId.OPTIMISM]: findDex('uniswapv3'),
+export const DEX_TO_COMPARE: { [chainId in ChainId]?: string } = {
+  [ChainId.BSCMAINNET]: 'pancake',
+  [ChainId.MATIC]: 'quickswap',
+  [ChainId.AVAXMAINNET]: 'traderjoe',
+  [ChainId.MAINNET]: 'uniswapv3',
+  [ChainId.FANTOM]: 'spookyswap',
+  [ChainId.CRONOS]: 'vvs',
+  [ChainId.AURORA]: 'trisolaris',
+  [ChainId.ARBITRUM]: 'sushiswap',
+  [ChainId.VELAS]: 'wagyuswap',
+  [ChainId.OASIS]: 'valleyswap-v2',
+  [ChainId.OPTIMISM]: 'uniswapv3',
```

Comment:
you can hardcode dexName here for displaying. 
ex: [ChainId.OPTIMISM]: {id: 'uniswapv3', name: "UniswapV3"}

Reply:
I will take it's name from ks-setting